src/index.js

```diff
@@ -100,6 +120,12 @@ export function filterCards(cards, options) {
 }
 
 export function deckFromCards(cards, images, options) {
+	// load syntax-highlighting css
+	let css = fs.readFileSync(path.resolve(__dirname, `../node_modules/@highlightjs/cdn-assets/styles/${options.codeStyle}.min.css`),'utf8');
```

Comment:
```suggestion
	let css = fs.readFileSync(path.resolve(__dirname, `../../node_modules/@highlightjs/cdn-assets/styles/${options.codeStyle}.min.css`),'utf8');
```

Rather strangely, I need to do `../../` in order to get this to work locally.

Reply:
Turns out I fucked this up by suggesting to remove the first element in the filedirname array. 
Because the first array entry always ends up being the *filename*, not the *dirname*

Reply:
You're absolutely right. I probably didn't notice because of that weird anki css caching.